bin/parse_phenotype.py

```diff
@@ -37,6 +37,24 @@
 assessments = ["BPQ", "TAS", "AQ", "EQ", "STAI",
     "GAD7", "MAIA", "PANAS", "PHQ9", "UCLA", "GSQ"]  # known assessments
 
+BL_one = ["participant_id", "BL_Track_meanacc", "BL_Track_RHO", "BL_Track_meanconf", "BL_Time_meanacc", "BL_Time_RHO",
+       "BL_Time_meanconf", "BL_Disc_meanacc", "BL_Disc_ROC", "BL_Disc_meanconf", "BL_d_prime", "BL_ITPE_T", "BL_ITPE_D", BL_C]
+F_one = ["participant_id", "F_Track_meanacc", "F_Track_RHO", "F_Track_meanconf", "F_Time_meanacc", "F_Time_RHO",
+       "F_Time_meanconf", "F_Disc_meanacc", "F_Disc_ROC", "F_Disc_meanconf", "F_d_prime", "F_ITPE_T", "F_ITPE_D", F_C]
+
+BL_assessment = ["participant_id", "BL_prosody_acc", "BL_face_acc", "BL_face_with_text_acc", "BL_text_acc", BL_percent_positive, BL_percent_negative,
```

Comment:
Same issue as L41. Also we already have `assessments` as a variable name. It seems like these are not fitting with that variable so you created a new one. The naming however, is confusing. Have you talked to Lisa to find out what these variables are?

Reply:
Yes I have named these variable in the same manner as the ones previous, I just dont know what assessment it is. I will update this once i know what the assessment is (I will ask Lisa).